fix(curriculum): prevent hardcoding palidrome checker answers (#54816)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-palindrome-checker-project/build-a-palindrome-checker.md

assert.strictEqual(resultEl.innerText.trim().replace(/[.,?!]+$/g, '').toLowerCase(), 'five|\_/|four is not a palindrome');
```

> When the `#text-input` element contains a palindrome of random letters and it's reversal, the `#result` element 

Reviewer: ```suggestion
When the `#text-input` element contains a palindrome created from a random sequence of alphanumeric characters, the `#result` element 
```

> When the `#text-input` element contains a palindrome created from a random sequence of alphanumeric characters, the `#result` element 
> should correctly identify it as a palindrome. 

Reviewer 2: ```suggestion
When the `#text-input` element contains an alphanumeric palindrome, the `#result` element should correctly identify it as a palindrome. 
```
Is it needed to mention how this palindrome was created?

Author: I guess not.
